src/OSPSuite.Infrastructure.Import/Core/DataSourceFileReaders/CsvDataSourceFile.cs

```diff
@@ -59,7 +59,7 @@ protected override void LoadFromFile(string path)
                }
 
                dataSheet.RemoveEmptyColumns();
-               dataSheet.RemoveEmptyRows();
+               dataSheet.RemoveEmptyRowsAtTheEnd();
```

Comment:
What happens if we replace this call with the new call to remove all rows?
Would we need these as separate steps?

Reply:
yeah, it's removing any blank rows at the end, but it seems that it might be also ok to just remove intermediate blank rows as well. Can you try it?